dockerfiles/theia/build.sh

```diff
@@ -22,9 +22,9 @@ tar rf ${DIR}/asset-che-theia.tar .git-che-theia-sha1
 gzip -f ${DIR}/asset-che-theia.tar
 
 # Download plugins
-THEIA_YEOMAN_PLUGIN="${DIR}/asset-untagged-c11870b25a17d20bb7a7-theia_yeoman_plugin.theia"
+THEIA_YEOMAN_PLUGIN="${DIR}/asset-untagged-8a7963262e021dab8ae0-theia_yeoman_plugin.theia"
 if [ ! -f "${THEIA_YEOMAN_PLUGIN}" ]; then
-    curl -L -o ${THEIA_YEOMAN_PLUGIN} https://github.com/eclipse/theia-yeoman-plugin/releases/download/untagged-c11870b25a17d20bb7a7/theia_yeoman_plugin.theia
+    curl -L -o ${THEIA_YEOMAN_PLUGIN} https://github.com/eclipse/theia-yeoman-plugin/releases/download/untagged-8a7963262e021dab8ae0/theia_yeoman_plugin.theia
```

Comment:
fyi @nickboldt 
I'm not sure, maybe it affects CRW built

Reply:
It's just another file

Reply:
should be OK in downstream but thx for heads up.

Are you cherrypicking this to 7.9.x branch too?

Reply:
If it's needed, I can do it

Reply:
LMK if you decide it's important. Plesae do it before EOD on Tuesday so we can include in 7.9.2. Please also add a link to the issue on https://github.com/eclipse/che/issues/16229 so we can track it.

Reply:
I'd say yes. We fixed the generator two months ago, but still don't use it.